.github/workflows/conda-package.yml

```
@@ -36,8 +36,14 @@ env:
       third_party/cupy/manipulation_tests/test_join.py
       third_party/cupy/manipulation_tests/test_rearrange.py
       third_party/cupy/manipulation_tests/test_transpose.py
+      third_party/cupy/math_tests/test_arithmetic.py
       third_party/cupy/math_tests/test_explog.py
+      third_party/cupy/math_tests/test_floating.py
+      third_party/cupy/math_tests/test_hyperbolic.py
+      third_party/cupy/math_tests/test_matmul.py
       third_party/cupy/math_tests/test_misc.py
+      third_party/cupy/math_tests/test_rounding.py
+      third_party/cupy/math_tests/test_sumprod.py
       third_party/cupy/math_tests/test_trigonometric.py
```

I believe it will work also:
```suggestion
      third_party/cupy/math_tests
```